projects/2.1/src/styles/components.scss

```diff
@@ -0,0 +1,130 @@
+@import 'base/api.scss';
+.app-social {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    font-size: $font-sm-small;
+    color: $banner-color;
+    i {
+        min-width: 31px;
+        height: 31px;
+        line-height: 31px;
+        border-radius: 50%;
+        background-color: $header-text-color;
+        text-align: center;
+        transition-duration: 0.5s;
+        cursor: pointer;
+        &:nth-child(2) {
+            margin-right: 10px;
+            margin-left: 10px;
+        }
+        &:hover {
+            color: $header-text-color;
+            background-color: $banner-color;
+            transition-duration: 0.5s;
+        }
+    }
+    &-header {
+        padding-top: 15px;
+        @include mq('sm') {
+            justify-content: flex-end;
+            padding-top: 0;
+        }
+    }
+    &-footer {
+        margin-top: 7px;
+        @include mq('sm') {
+            margin-top: 0;
+        }
+        i {
+            color: $header-text-color;
+            background-color: $banner-color;
+            &:hover {
+                color: $banner-color;
+                background-color: $header-text-color;
+            }
+        }
+    }
+}
+
+.info-days__number {
+    color: $text-color;
+    font: {
+        family: 'Trebuchet MS Regular' $font-fallback;
+        size: 80px;
+        weight: 400;
+    }
+    line-height: 100px;
+    @include mq('sm') {
+        font-size: 100px;
+    }
+}
+
+.info-days__subtitle {
+    text-align: justify;
+    font: {
+        size: 29px;
+        weight: 400;
+    }
+    line-height: 32px;
+    width: 160px;
+}
+
+.app-title {
+    font-size: $font-sm-small;
+    font-weight: 700;
+    @include mq('lg') {
+        font-size: $font-md-small;
+    }
+}
+
+.offer-title {
+    border-bottom: 1px solid #7f7f7f;
+    margin: {
+        bottom: 0;
+    }
+}
+
+.number-title {
+    font-weight: 400;
+    line-height: 30px;
+    margin: {
+        top: 5px;
+        bottom: 20px;
+    }
+}
+
+.app-text {
+    font: {
+        family: 'Arial Regular' $font-fallback;
+        size: $font-xs-small;
+        weight: 400;
+    }
+    line-height: 20px;
+    margin-bottom: 27px;
+    text-align: justify;
+    @include mq('sm') {
+        text-align: left;
+    }
+    &--bold {
+        font: {
+            weight: 700;
+            size: 16px;
+        }
+        text-align: justify;
```

Comment:
app-text only styles markup, text-align make text-position, better to use this prop in concrete markup selector